scripts/bootstrap.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+const inquirer = require('inquirer');
+const program = require('commander');
+const childProcess = require('child_process');
+const chalk = require('chalk');
+const log = require('npmlog');
+
+const { lstatSync, readdirSync } = require('fs');
+const { join } = require('path');
+
+const isTgz = source => lstatSync(source).isFile() && source.match(/.tgz$/);
+const getDirectories = source => readdirSync(source).map(name => join(source, name)).filter(isTgz);
+
+log.heading = 'storybook';
+const prefix = 'bootstrap';
+
+const spawn = command =>
+  childProcess.spawnSync(`${command}`, {
+    shell: true,
+    stdio: 'inherit',
+  });
+
+const main = program
+  .version('3.0.0')
+  .option('--all', `Bootstrap everything ${chalk.gray('(all)')}`);
+
+const createTask = ({ defaultValue, option, name, check = () => true, command, pre = [] }) => ({
+  value: false,
+  defaultValue: defaultValue || false,
+  option: option || undefined,
+  name: name || 'unnamed task',
+  check: check || (() => true),
+  command: () => {
+    // run all pre tasks
+    // eslint-disable-next-line no-use-before-define
+    pre.map(key => tasks[key]).forEach(task => {
+      if (!task.check()) {
+        task.command();
+      }
+    });
+
+    log.info(prefix, name);
+    command();
+  },
+});
+
+const tasks = {
+  reset: createTask({
+    name: `Clean and re-install root dependencies ${chalk.red('(reset)')}`,
+    defaultValue: false,
+    option: '--reset',
+    command: () => {
+      log.info(prefix, 'git clean');
+      spawn('git clean -fdx --exclude=".vscode" --exclude=".idea"');
```

Comment:
State exactly what will be removed: all the files not present in git, except for .idea and .vscode. This is more verbose, but also more fair